qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -673,13 +673,21 @@ def _decomposer_2q_from_target(self, target, qubits, approximation_degree):
         # available instructions on this qubit pair, and their associated property.
         available_2q_basis = {}
         available_2q_props = {}
+
```

Comment:
Can you add a comment on why we're making this replacement?

Reply:
@kdk and @ewinston Thanks for looking at this. I added a bit of explanation in the initial comment in this PR.

Reply:
I think an inline comment here would be useful too. Just because the selection logic here is a bit dense already and this adds a new wrinkle to it.